src/repl/scala/tools/nsc/interpreter/ReplProps.scala

```diff
@@ -32,6 +32,12 @@ class ReplProps {
   val powerInitCode   = Prop[JFile]("scala.repl.power.initcode")
   val powerBanner     = Prop[JFile]("scala.repl.power.banner")
 
+  val prompt = Prop[String]("scala.shell.prompt")
+  val prompt2 = Prop[String]("scala.shell.prompt2")
```

Comment:
I thought that "suffix" 2 is sensible (similar to PS2 in Unix shell)
Let me know what do you propose ?